scripts/lib/upgrade-zulip-stage-2

```diff
@@ -104,7 +104,7 @@ def shutdown_server() -> None:
         core_server_services.append("zulip-thumbor")
 
     logging.info("Stopping Zulip...")
-    subprocess.check_call(["supervisorctl", "stop", *core_server_services, *worker_services],
+    subprocess.check_call(["supervisorctl", "stop", *worker_services, *core_server_services],
```

Comment:
I think we want to shell out twice; I think `supervisord` will just try to stop them all at once.

(In any case smaller batches tend to work better with supervisord restarts, especially if you can have the service be user-facing up during the first batch)

Reply:
[`supervisorctl start` treats its arguments sequentially](https://github.com/Supervisor/supervisor/blob/master/supervisor/supervisorctl.py#L782), as I noted in the commit message -- I'll add a citation for that to the source code.

[`supervisorctl restart` is exactly equivalent to stop/start in the same order](https://github.com/Supervisor/supervisor/blob/master/supervisor/supervisorctl.py#L967-L968); and [`stop` also treats its arguments sequentially](https://github.com/Supervisor/supervisor/blob/master/supervisor/supervisorctl.py#L859).

We can split this out into separate calls if you'd prefer, but this seems like this behaviour is unlikely to change in supervisor.  AFAICT the calls into e.g. `stopProcessGroup` and their results are synchronous, particularly since we're printing the results right there, so I don't believe that any of the starting or stopping overlaps as it iterates through its arguments.